recipes/pcapplusplus/cmake/conanfile.py

```diff
@@ -0,0 +1,60 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.files import copy, get, rmdir
+from conan.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.57.0"
```

Comment:
```suggestion
required_conan_version = ">=1.52.0"
```

1.52.0 is sufficient (even less maybe, but there are some tedious issues before 1.52.0, so I think this version is fine)

Reply:
Fixed in https://github.com/conan-io/conan-center-index/pull/20296/commits/fe96dbda54ca1e70a1960957a445a4f6c080dfc3